docs/imagecustomizer/concepts/iso.md

```diff
@@ -4,7 +4,7 @@ title: ISO Support
 nav_order: 2
 ---
 
-# Image Customizer ISO Support
+# Image Customizer LiveOS ISO Support
```

Comment:
Do we want to update it to Prism thoroughly?

Reply:
I don't want to be inconsistent with the names. So, I'll let the conversion happen in a single batch.